src/sgame/sg_bot_util.cpp

```diff
@@ -2493,6 +2493,17 @@ static gentity_t *BotPopEnemy( enemyQueue_t *queue )
 
 void BotPain( gentity_t *self, gentity_t *attacker, int )
 {
+	// alien bots should not chase human players when attacked by turrets
+	if ( G_Team( self ) == TEAM_ALIENS
+		 && G_Team( attacker ) != TEAM_NONE
+		 && attacker->s.modelindex == BA_H_MGTURRET
+		 && self->botMind->goal.targetsValidEntity()
+		 && self->botMind->goal.getTargetedEntity()->s.eType == entityType_t::ET_PLAYER )
```

Comment:
Do you think there should be some time of heuristic for tradeoff between pursuing into a base (ie, human attacker equipment > bot value) or is this sufficient.

Can this be abused by players by simply building a turret to have bots discourage people from attacking?

Lastly, should this consider rocket pods as well, or is the range too great for this to be effective?

Reply:
A better heuristic can never hurt. The current heuristic is in `BotFindBestEnemy`.

I assume it can be abused. But the current situation can be abused even more. Build a wall of turrets, make an alien bot chase you, run to the wall and jump over it. The bots will not even try to attack the turrets. A tyrant will get stuck there, because it cannot jump over.

Rocket pod range is too high for a simple solution.